code/modules/projectiles/gun.dm

```diff
@@ -896,8 +896,8 @@
 				"desc" = F.desc,
 				"burst" = F.settings["burst"],
 				"minimum_fire_delay" = GUN_MINIMUM_FIRETIME,
-				"fire_delay" = F.settings["fire_delay"],
-				"move_delay" = F.settings["move_delay"],
+				"fire_delay" = F.settings["fire_delay"] * 5,
+				"move_delay" = F.settings["move_delay"] * 5,
```

Comment:
Why the `* 5` ?

Reply:
It calls ticks ms, but ticks are not real, so I used fortune telling to determine how many ms one tick will be during launch.